Jenkinsfile

```diff
@@ -52,12 +60,45 @@ def presubmit(gitUtils, bazel, utils) {
     }
     stage('Integration Tests') {
       timeout(15) {
-        sh('bin/e2e.sh -tag alpha' + gitUtils.GIT_SHA + ' -v 2')
+        sh("bin/e2e.sh -tag alpha${gitUtils.GIT_SHA} -v 2")
+      }
+    }
+    stage('Build istioctl') {
+      def remotePath = gitUtils.artifactsPath('istioctl')
+      sh("bin/cross-compile-istioctl -p ${remotePath}")
+    }
+  }
+}
+
+def stablePresubmit(gitUtils, bazel, utils) {
+  goBuildNode(gitUtils, 'istio.io/manager') {
+    bazel.updateBazelRc()
+    utils.initTestingCluster()
+    sh('ln -s ~/.kube/config platform/kube/')
+    stage('Integration Tests') {
+      timeout(30) {
+        sh("bin/e2e.sh -count 10 -debug -tag debug${gitUtils.GIT_SHA} -v 2")
```

Comment:
I have to qualify on something. If the test fail then the next commit will be qualified. We don t have to qualify every commit.